linera-client/src/persistent/file.rs

```diff
@@ -69,7 +69,7 @@ impl Lock {
 
 impl Drop for Lock {
     fn drop(&mut self) {
-        if let Err(error) = self.0.file().unlock() {
+        if let Err(error) = fs4::FileExt::unlock(self.0.file()) {
```

Comment:
nit: Should we have `use fs4::FileExt` above and write `FileExt::unlock()` then?

Reply:
Indeed! Good catch, thanks!